.circleci/config.yml

```diff
@@ -354,7 +354,7 @@ jobs:
   build-examples:
     machine:
       image: ubuntu-1604:202007-01
-    resource_class: medium
+    resource_class: large
```

Comment:
is medium too slow?

Reply:
yes it is. Thats why i changed it to large